tests/Basic/ArtworkTest.php

```diff
@@ -27,4 +29,83 @@ protected function tearDown(): void
 
         parent::tearDown();
     }
+
+    /** @test */
+    public function it_fetches_the_gallery_for_an_artwork(): void
+    {
+        $gallery = $this->make(Gallery::class, ['is_closed' => false]);
+        $galleryKey = $gallery->getAttributeValue($gallery->getKeyName());
+
+        $artwork = $this->make(Artwork::class, ['gallery_id' => $galleryKey, 'is_on_view' => true]);
+        $artworkKey = $artwork->getAttributeValue($artwork->getKeyName());
+
+        $response = $this->getJson('api/v1/artworks/' . $artworkKey);
+        $response->assertSuccessful();
+
+        $resource = $response->json()['data'];
+        $this->assertTrue($resource['is_on_view']);
+
+        Artwork::query()->delete();
+        Gallery::query()->delete();
+    }
+
+    /** @test */
+    public function it_fetches_artwork_linked_art_endpoint(): void
+    {
+        $artworkType = $this->make(ArtworkType::class, ['aat_id' => '300033618', 'title' => 'Painting']);
+        $artworkTypeKey = $artworkType->getAttributeValue($artworkType->getKeyName());
+
+        $artwork = $this->make(Artwork::class, ['artwork_type_id' => $artworkTypeKey]);
+        $artworkKey = $artwork->getAttributeValue($artwork->getKeyName());
+
+        $response = $this->getJson('la/v1/objects/' . $artworkKey);
+        $response->assertSuccessful();
+
+        $resource = $response->json();
+        $this->assertCount(1, $resource['classified_as']);
+        $this->assertEquals($resource['classified_as'][0]['id'], 'http://vocab.getty.edu/aat/300033618');
+
+
+        Artwork::query()->delete();
+        ArtworkType::query()->delete();
+    }
+
+    /** @test */
+    public function it_fetches_multiple_artwork_linked_art_endpoint(): void
+    {
+        $artwork1Type = $this->make(ArtworkType::class, ['aat_id' => '300033618', 'title' => 'Painting']);
+        $artwork1TypeKey = $artwork1Type->getAttributeValue($artwork1Type->getKeyName());
+
+        $artwork1 = $this->make(Artwork::class, ['artwork_type_id' => $artwork1TypeKey]);
+        $artwork1Key = $artwork1->getAttributeValue($artwork1->getKeyName());
+
+        $artwork2Type = $this->make(ArtworkType::class, ['aat_id' => '300193015', 'title' => 'Vessel']);
+        $artwork2TypeKey = $artwork2Type->getAttributeValue($artwork2Type->getKeyName());
+
+        $artwork2 = $this->make(Artwork::class, ['artwork_type_id' => $artwork2TypeKey]);
+        $artwork2Key = $artwork2->getAttributeValue($artwork2->getKeyName());
+
+        $response = $this->getJson('la/v1/objects?ids=' . $artwork1Key . ',' . $artwork2Key);
+        $response->assertSuccessful();
+
+        $resource = $response->json();
+        $this->assertCount(2, $resource['objects']);
+
+        // Check that the works we added to the database are in the repsonse
```

Comment:
For most assertions, you can pass in a final `$message` parameter that describes the assertion in more detail, and prints that message out on failure:
https://docs.phpunit.de/en/10.1/assertions.html#assertcontains

I recommend moving these types of comments into that message parameter, for example:
```php
$this->assertContains(
    $artwork1Key,
    $idsInResponse,
    "The works we added to the database are in the response"
);
```